spec/serializers/document_serializer_spec.rb

```diff
@@ -7,24 +7,26 @@
 
   let(:document) { create :document }
   let(:result) { JSON.parse(ActiveModelSerializers::Adapter.create(serializer).to_json).deep_symbolize_keys }
+  let(:result_data) { result[:data] }
+  let(:attributes) { result_data[:attributes] }
```

Comment:
The use of nested intermediate variables here smells a bit to me because it obscures the shape of the result hash in the `expect` statements. I'm not tied to this point, but I'm not sure there's a meaningful benefit to deduplicating those 3 calls to attributes (and 5 to data) that's worth the cost of the added complexity.